src/openwith.js

```diff
@@ -16,21 +16,31 @@
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 //
 OME.setOpenWithUrlProvider("omero_iviewer", function(selected, url) {
+    selected_count = selected.length;
+    // last selected image
+    last_selected = selected[selected_count-1];
     // Add image Id to url
-    url += selected[0].id + "/";
+    url += last_selected.id + "/";
```

Comment:
this just determines the image that is opened by default. 

things would have to be changed in a different place, the rationale for the existing solution was mainly that one would have to expand downwards (with more than 10 images) when the first image one sees is the last one.